lodash.js

```diff
@@ -17020,9 +17020,10 @@
     baseForOwn(LazyWrapper.prototype, function(func, methodName) {
       var lodashFunc = lodash[methodName];
       if (lodashFunc) {
-        var key = (lodashFunc.name + ''),
-            names = realNames[key] || (realNames[key] = []);
+		var key = lodashFunc.name + '';
+		if (!realNames.hasOwnProperty(key)) return;
 
```

Comment:
☝️ indentation nits. Also you can use the local `hasOwnProperty.call(realNames, key)`

Reply:
do I need to resubmit the PR?

Reply:
Nope. You can make changes `git add` the files then `git commit --amend` then `git push origin +realNames-ownProperty` (force push an update)

Reply:
done